en/index.rst

```diff
@@ -33,9 +33,9 @@ Mapping Objects onto a Document Repository
   :doc:`Hierarchies and References <reference/association-mapping>` |
   :doc:`Inheritance <reference/inheritance-mapping>`
 
-* **Mapping Driver References**: (TODO: build this in a sane way, without duplicating too much information)
+* **Mapping Driver References**:
   :doc:`Docblock Annotations <reference/annotations-reference>` |
-  :doc:`XML <reference/xml-mapping>` |
+  :doc:`XML <reference/xml-reference>` |
```

Comment:
i think calling this xml-mapping would be more correct, as it is not a reference.